app/javascript/src/BackendApis/components/NewBackendModal.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react'
+
```

Comment:
For consistency with `NewBackendForm.jsx`, you coud import `useState` and `useEffect`